test/Microsoft.AspNetCore.DataProtection.Extensions.Test/DataProtectionProviderTests.cs

```diff
@@ -38,6 +38,46 @@ public void System_UsesProvidedDirectory()
         [ConditionalFact]
         [ConditionalRunTestOnlyIfLocalAppDataAvailable]
         [ConditionalRunTestOnlyOnWindows]
```

Comment:
I don't think that matters because there is no other test that can access this folder.